CMakeLists.txt

```diff
@@ -206,6 +210,20 @@ endif()
 message(STATUS
     "Compiled IDs C:${CMAKE_C_COMPILER_ID}, C++:${CMAKE_CXX_COMPILER_ID}")
 
+# Always disable SSSE3 since it is rare to have SSSE3 but not SSE4
+set(HWY_DISABLED_TARGETS "HWY_SSSE3")
```

Comment:
Doesn't this override user settings?

Reply:
I guess it does, but we are already doing that anyway in our current build script: https://github.com/libjxl/libjxl/blob/main/CMakeLists.txt#L304